.github/workflows/notify-on-issue-and-release.yml

```diff
@@ -0,0 +1,22 @@
+name: 'notify-on-issue-and-release'
+
+on:
+  issues:
+    types: ['opened']
+  release:
+    types: ['released']
+  workflow_dispatch:
+
+jobs:
+  call_action:
+    runs-on: 'ubuntu-latest'
+    permissions:
+      contents: 'read'
+      id-token: 'write'
+    steps:
+      - name: 'Checkout'
+        uses: 'actions/checkout@8f4b7f84864484a7bf31766abe9204da3cbe65b3' # ratchet:actions/checkout@v3
+      - name: 'call action'
+        uses: 'google-github-actions/send-google-chat-webhook@v0.0.2'
```

Comment:
Should we pin this to a specific commit, similarly to the checkout action at line 18? That would also save us from waiting for a release of the google-github-actions repo.

Reply:
The pin doesn't work, cause the action downloads go binary base on the release tag and run that binary, currently v0.0.1 doesn't support issue and release, and v0.0.2 is not released yet.

Reply:
Ah, I see.